longevity_test.py

```diff
@@ -154,6 +154,13 @@ def run_prepare_write_cmd(self):
             for stress in verify_queue:
                 self.verify_stress_thread(cs_thread_pool=stress)
 
+        prepare_end_cql_cmds = self.params.get('prepare_end_cql_cmds')
+        if prepare_end_cql_cmds:
+            self._run_cql_commands(prepare_end_cql_cmds)
+
+        prepare_compaction_wait_time = self.params.get('prepare_compaction_wait_time')
```

Comment:
Why not just putting a number here that would be big enough?
It looks like over using of parameters that make the code more and more complex.

Reply:
Hardcoding it, so all longevity will wait for compaction to settle down ? Are you sure ?

What if we have nemesis during prepare ? 

I thought to do it minimal as possible to avoid breaking other longevity

I agree we have lots of config parameters, that are lots of think we want to be able to configure...